docs/migrating/migrating_from_ez_publish_platform.md

```diff
@@ -196,6 +196,40 @@ Steps here should only be done once you are ready to move away from legacy and L
 
 The command example suggests using the verbose flag. This is optional, but recommended as we are actively gathering feedback on how it works with older eZ Publish content, and on how we can improve it both [via issues in the repository](https://github.com/ezsystems/ezplatform-xmltext-fieldtype), and on Slack.
 
+Also, note that if you have made custom image classes, you must also provide their content class identifiers:
+
+`php app/console ezxmltext:convert-to-richtext --image-content-types=image,thumbnail,profile -v`
+
+The script needs to know these identifiers in order to convert `<ezembed>` tags correctly. Failing to do so will prevent the editor from showing image thumbnails of embedded image objects. You may find the image Content Types in your installation by looking for these settings in `content.ini(.append.php)`:
+
+```
+[RelationGroupSettings]
+ImagesClassList[]
+ImagesClassList[]=image
+```
+
+!!! note
+
+    Version of the migration script prior to v1.5.1 would fail to convert embedded images correctly. If you have a database which you have already converted with an old version, you may rerun the `convert-to-richtext` command with the following options:
```

Comment:
`v1.5.1` => `ezplatform-xmltext-fieldtype v1.6.0`  

1. to be more specific
2. as we are talking about prior (1.5.1 is also affected)

Reply:
Fixed